boards/arm/qemu_cortex_a53_zynqmp/qemu_cortex_a53_zynqmp.yaml

```diff
@@ -0,0 +1,13 @@
+identifier: qemu_cortex_a53_zynqmp
+name: QEMU Emulation for Xilinx ZynqMQ APU
+type: qemu
+simulation: qemu
+arch: arm
+toolchain:
+  - zephyr
+  - cross-compile
+ram: 128
```

Comment:
Cortex-A53 on ZCU102 has access to 2GB RAM. Are we purposely limiting this number ?

Reply:
This is intended to be CI testing-only platform and we are not going to have 2GB RAM for it (e.g. see `qemu_cortex_r5` FDT).

That aside, the RAM size here would need to be updated indeed, once we finalised the RAM size in the FDT.